subsys/mpsl/mpsl_init.c

```diff
@@ -9,13 +9,54 @@
 #include <kernel.h>
 #include <mpsl.h>
 
-#define MPSL_LOW_PRIO_IRQ SWI4_IRQn
+#define MPSL_LOW_PRIO_IRQn SWI4_IRQn
```

Comment:
Not related to your change. Is there a reason for using SWI4 instead of SWI5? For the ble controller we use SWI5

Reply:
Not anymore. Initially it was to make MPSL clock driver work with Zephyr.